debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/FieldSelector.java

```diff
@@ -116,12 +116,12 @@ public FieldSelector build() {
 
         private void parse(String value, Function<String, Path> pathCreator, List<Path> result) {
             if (!Strings.isNullOrEmpty(value)) {
-                Arrays.stream(value.split(",")).forEach(name -> result.add(pathCreator.apply(name)));
+                Arrays.stream(value.trim().split(",")).forEach(name -> result.add(pathCreator.apply(name)));
```

Comment:
Very nice; great to see this change, as I was expecting the new test to run into this. Thanks!